Packs/Synapse/Integrations/Synapse/README.md

```diff
@@ -0,0 +1,736 @@
+Synapse intelligence analysis platform.
+This integration was integrated and tested with version xx of Synapse
```

Comment:
Last thing - could you please update what version of Synapse you worked on?

Reply:
@orlichter1  - updated version in `README.md` as requested and also bumped to latest version of docker image `demisto/aiohttp:1.0.0.12410`.